_posts/2021-02-15-contributing-to-solidity-101.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+published: true
+title: 'Contributing to Solidity 101'
+date: '2021-02-15'
+author: Franziska Heintel
+category: Explainers
+---
+
+The Solidity programming language is an open-source project governed by a core team. We rely on the community’s feedback, input and contributions to make the language as effective, safe and useful as possible. 
+
+In this post, we will walk you through the various different ways how you can get involved contributing to Solidity! Do not hesitate to contact us in case anything is left unclear after reading the post.
+
+### Overview of Contribution Opportunities  
+
+There are plenty of options how you can contribute to Solidity.  
+
+In particular, we appreciate support in the following areas:
+
+- **Issues**: Help with reporting or fixing issues.
+- **Testing**: Run the compiler tests or help with fuzz testing.
```

Comment:
Ah, now I understand! I guess running the compiler tests is not that helpful, but probably something like "testing out new compiler versions, preferrably nightly versions, on your code".

Reply:
Ok perfect, will add that (and remove the "run the tests bit")! What about helping with fuzzing? https://docs.soliditylang.org/en/v0.8.1/contributing.html#running-the-fuzzer-via-afl